### ISSUE TITLE
PlayerView#onTouchEvent() should return false if useController is false

### ISSUE THREAD

**Issue by AlinaVoronkovaEpam** [google/ExoPlayer#6340]
### Issue description
In our app we do not use a player controller and disable it this way: 
```
view.setUseController(false)
```
However, the touch event is consumed by the view.

### Reproduction steps
I'm not allowed to share the app code.
Please see the code snippet `com.google.android.exoplayer2.ui.PlayerView` (ExoPlayer ver. 2.10.2) below that demonstrate the touches are consumed by the view in any cases:
```
  @Override
  public boolean onTouchEvent(MotionEvent event) {
    switch (event.getAction()) {
      case MotionEvent.ACTION_DOWN:
        isTouching = true;
        return true;
      case MotionEvent.ACTION_UP:
        if (isTouching) {
          isTouching = false;
          performClick();
          return true;
        }
        return false;
      default:
        return false;
    }
  }
``` 

### Link to test content
The issue is reproduced with any test assets.

### A full bug report captured from the device
A bug report has been sent by email.

### Version of ExoPlayer being used
The current ver is 2.10.2.
The issue is not seen with ExoPlayer 2.9.3.

### Device(s) and version(s) of Android being used
- Pixel 2 XL + Android 9 (test device)

The issue is reproduced on any Android devices.

**Comment by AlinaVoronkovaEpam** [comment]
Please take a look at exactly the same issue https://github.com/google/ExoPlayer/issues/1923
Thanks.

**Comment by marcbaechinger** [comment]
Can you please trying to reproduce with the most recent version of the release-v2 branch? It appears to me that this has been fixed already in [PlayerView](https://github.com/google/ExoPlayer/blob/release-v2/library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerView.java#L1056):

```
public boolean onTouchEvent(MotionEvent event) {
    if (!useController || player == null) {
      return false;
    }
    switch (event.getAction()) {
       // ...
```

Please let us know whether this solves your problem or we need to take a look.

**Comment by ojw28** [comment]
Yes, this was fixed in 2.10.3 as per the release notes (see also https://github.com/google/ExoPlayer/issues/6109).
